### ISSUE TITLE
Explore how emerging technologies could improve accessibility

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-frontend#3044]
## What
**Timeline:** Ongoing, as opportunities arise
**Priority level:** 6 – what is good to do, without adding risk to the priorities above

## Why
As the technology we use to deliver accessible services advances, these new technologies may offer new ways to improve on accessibility. Conversely, if technology advances and we don't work to keep up, we may find our styles, components and patterns becoming out of synch and less accessible over time.

Areas that are of particular interest for exploration include:
- Updates to assistive technologies we already test with or support
- Updates to browsers we already test with or support
- Updates to operating systems that have an impact on accessibility tools or features
- New tools for accessibility testing, either manual or automated

Areas we want to be cautious around include:
- Emerging technologies that promote themselves as being assistive technologies, but don't have evidence of real-world benefit or use for disabled people
- New or updated browsers or browser features that aren't within the scope of our browser support
- Any technology which out-of-the-box is less accessible than current options

### Potential tasks

- Research new assistive technology
- Research new browser, operating system and CSS features
- Avoid pursuing emerging technologies that lack evidence of real-world use or benefit for disabled people

## Who needs to work on this
Developers, accessibility specialist

## Who needs to review this
Product and delivery managers, developers

## Done when

- [ ] Thing to do
